les/handler.go

```diff
@@ -286,17 +283,8 @@ func (pm *ProtocolManager) handle(p *peer) error {
 		p.Log().Debug("Light Ethereum handshake failed", "err", err)
 		return err
 	}
-
-	if !pm.lightSync && !p.Peer.Info().Network.Trusted {
-		addr, ok := p.RemoteAddr().(*net.TCPAddr)
```

Comment:
So this piece of code existed to IIUC to assign some kind of reputation to IP addresses, because otherwise light clients could just create a new ID and further free-ride. I'm curious what led to the removal of this code, and if we have coverage for malbehavior instead?